main/src/ca/uwaterloo/flix/runtime/shell/ShellParser.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 Paul Butcher
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package ca.uwaterloo.flix.runtime.shell
+
+import org.jline.reader.{EOFError, ParsedLine, Parser}
+import org.jline.reader.Parser.ParseContext
+
+import scala.jdk.CollectionConverters._
+
+/**
+  * Minimal implementation of `ParsedLine`, necessary to keep jline happy
+  */
+class Parsed(s: String) extends ParsedLine {
+
```

Comment:
extra newline

Reply:
Forgive me, I'm not sure whether this is asking me to add an extra newline, or telling me that there's a newline here that you would like removed?

Reply:
Just noticed a stray newline.